src/sonic-config-engine/minigraph.py

```diff
@@ -1824,6 +1837,30 @@ def parse_asic_meta_get_devices(root):
 
     return local_devices
 
+def parse_meta_get_macsec_profile(root, hname):
```

Comment:
Can we add parsing of `macsec-profile` in this function  https://github.com/Azure/sonic-buildimage/blob/ab0930313b34aa37819b4a5a325ed8b2f633e028/src/sonic-config-engine/minigraph.py#L873?